source/includes/mongodb-compatibility-table-motor.rst

```diff
@@ -5,62 +5,129 @@
    :class: compatibility
 
    * - Motor (Python async) Driver
+     - MongoDB 4.4
+     - MongoDB 4.2
+     - MongoDB 4.0
      - MongoDB 3.6
      - MongoDB 3.4
      - MongoDB 3.2
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - 2.2
```

Comment:
Per https://github.com/mongodb/docs-ecosystem/pull/652/files#r454002968 I assume this row is pending removal.